Odata-docs/webapi-8/fundamentals/actions-functions.md

```diff
@@ -0,0 +1,423 @@
+---
+title: "Actions and functions"
+description: "This documentation describes how to add actions and functions."
+
+author: KenitoInc
+ms.author: kemunga
+
+ms.date: 9/15/2022
+---
+# Actions and functions
+**Applies To**:[!INCLUDE[appliesto-webapi](../../includes/appliesto-webapi-v8.md)]
+
+In OData, actions and functions are a way to add server-side behavior that are not easily defined as CRUD operations on entities. This documentation shows how to add actions and functions to an OData v4 endpoint, using ASP.NET Core OData v8.x.
+
+## What are actions and functions?
+- `Functions` are operations exposed by an OData service that MUST return data and MUST have no observable side effects. They may support further composition. Functions are invoked by using HTTP GET requests.
```

Comment:
I could be wrong, but I think we can invoke functions with `POST` requests, for example when passing parameters in the body if the inputs are complex objects.